src/library/scala/collection/immutable/List.scala

```diff
@@ -203,10 +205,14 @@ sealed abstract class List[+A] extends AbstractSeq[A]
     loop(null, null, this, this)
   }
 
+  private def isLikeListReusableCBF( bf: CanBuildFrom[_,_,_]): Boolean = {
+//    bf eq List.ReusableCBF
+    (bf eq List.ReusableCBF) ||(bf eq immutable.Seq.ReusableCBF) ||(bf eq collection.Seq.ReusableCBF)
```

Comment:
Do we have a test that would remind us to change it is, for example, `immutable.Seq()` were changed to return `Vector` rather than `List`? I know that's something of a hypothetical but it would be easy enough to defend with a test.

Reply:
test added